### ISSUE TITLE
no wrap around when "jump_to_most_recent_on_show": false,

### ISSUE THREAD

**Issue by reagle** [kapitanluffy/sublime-context-keeper#7]
When `"jump_to_most_recent_on_show": false,` I noticed that when I'm on the last tab I can not move forward and wrap around to the first tab again. I'm not sure if ST does this, but I was expecting it to. Perhaps my expectation was off?

**Comment by kapitanluffy** [comment]
Yep unfortunately ST does not provide this function :(. I am still yet to find a hacky way around this
